src/Wordpress/DataStore.php

```diff
@@ -70,4 +73,46 @@ public function getCloudFlareEmail()
     {
         return get_option(self::EMAIL);
     }
+
+    /**
+     * @return (bool)
+     */
+    public function getPluginSettings($api)
+    {
+        $ip_rewrite_value = get_option(self::CLOUDFLARE_SETTING_PREFIX + self::IP_REWRITE);
+        $protocol_rewrite_value = get_option(self::CLOUDFLARE_SETTING_PREFIX + self::PROTOCOL_REWRITE);
+
+        $settings = [];
+        array_push($settings, $api->createPluginResult(self::IP_REWRITE, $ip_rewrite_value, true, ''));
+        array_push($settings, $api->createPluginResult(self::PROTOCOL_REWRITE, $protocol_rewrite_value, true, ''));
+
```

Comment:
Yes!  This is exactly what I was thinking - nice job.  Very clean, easily extendible in the future.